.github/workflows/lint.yml

```diff
@@ -0,0 +1,14 @@
+name: Lint
+
+on: [push, pull_request]
```

Comment:
Same goes for here (original comment [here](https://github.com/Kong/kong-plugin-openwhisk/pull/21/files#r997201670))